src/monitoring/monitoring.service.ts

```diff
@@ -16,6 +16,7 @@ let versionCache: null | {
   preRelease?: string;
   commit?: string;
 } = null;
+
```

Comment:
stray newline?

Reply:
Yeah it's probably added by changing something and changing back again.